src/__tests__/plugin.js

```diff
@@ -109,4 +109,35 @@ describe('gatsby-remark-embedder', () => {
 
     expect(remark.stringify(processedAST)).toMatchSnapshot();
   });
+
+  it('can execute custom transformers', async () => {
+    const markdownAST = getMarkdownASTForFile('CustomTransformer');
+
+    const transformer = {
+      shouldTransform: jest.fn(url => url.startsWith('https://some-site.com')),
+      getHTML: jest.fn(url => `<iframe href="${url}"></iframe>`),
+    };
+
+    const processedAST = await plugin(
+      { cache, markdownAST },
+      { customTransformers: [transformer] }
+    );
+
+    expect(transformer.shouldTransform).toHaveBeenCalledTimes(2);
+    expect(transformer.shouldTransform).toHaveBeenCalledWith(
+      'https://some-site.com/id/abc'
+    );
+    expect(transformer.shouldTransform).toHaveBeenCalledWith(
+      'https://some-other-site.com/'
```

Comment:
```suggestion
      'https://some-other-site.com'
```

Reply:
I cannot change the expectation since the `getUrlString` function adds the trailing slash.

This happens because the `URL` object adds it:

```js
new URL('http://some-other-site.com').toString() === 'http://some-other-site.com/'
```